model/pipeline/src/main/proto/beam_runner_api.proto

```diff
@@ -228,9 +228,9 @@ message StandardPTransforms {
     // Payload: CombinePayload
     COMBINE_GLOBALLY = 1 [(beam_urn) = "beam:transform:combine_globally:v1"];
 
-    // Represents the Combine.groupedValues() operation.
-    // If this is produced by an SDK, it is assumed that the SDK understands
-    // each of CombineComponents.
+    // Represents the Combine Grouped Values transform, as described in the
+    // following document:
+    // https://s.apache.org/beam-runner-api-combine-model#heading=h.aj86ew4v1wk
```

Comment:
Should this be moved from `Composites` to `CombineComponents`?

Reply:
Yeah, that seems like a good idea. Is it as easy as just deleting it and changing the numbers on the existing enum values, or would that cause issues?